### DISCUSSION TITLE
An error occur when i run the example

### DISCUSSION THREAD

**Discussion by xuexl** [neuronika/neuronika#99] [General]
Hi, I am a beginner of rust, when i run the example code of "quickstart" that is in the directory of examples, an error occur, the error information is :

    error[E0277]: the trait bound NeuralNetwork: serde::de::Deserialize<'_> is not satisfied
    --> src/main.rs:51:21
    |
    51 | let json_data = serde_json::from_str(
    | ^^^^^^^^^^^^^^^^^^^^ the trait serde::de::Deserialize<'_> is not implemented for NeuralNetwork
    |
    note: required by a bound in serde_json::from_str
    --> /home/lin/.cargo/registry/src/github.com-1ecc6299db9ec823/serde_json-1.0.79/src/de.rs:2611:8
    |
    2611 | T: de::Deserialize<'a>,
    | ^^^^^^^^^^^^^^^^^^^ required by this bound in serde_json::from_str

I know the mean, but need i to write the code of "Deserialize trait" or , maybe, the reason is the error config of my toml file.

The content of my toml file is :

[dependencies]
neuronika = { git = "https://github.com/neuronika/neuronika" }
csv = "1.1.6"
itertools = "0.10.3"
ndarray = {version = "0.15.4", features = ["rayon"]}
ndarray-rand = "0.14.0"
rand = "0.8.4"
rand_distr = "0.4.2"
rayon = "1.5.1"
serde = {version = "1.0.136", features = ["derive"]}
serde_json = "1.0.79"

[features]
blas = ["ndarray/blas"]
matrixmultiply-threading = ["ndarray/matrixmultiply-threading"]
serialize = ["ndarray/serde"]

Help me, please. Thank you .

**Comment by frjnn** [comment]
Answered in #97.
